blog/2025-10-08-python-314.md

```diff
@@ -0,0 +1,53 @@
+
+With yesterday's release of Python 3.14 (insert release link), we not only have Python 3.14 itself available on conda-forge,
+but also a wide selection of packages to make use of it.
+As the conda ecosystem is based on binary packages, this means that several packages are already built for Python 3.14.
+
+You can create a new environment by using:
+
+```
+conda create -n py314 python=3.14 -c conda-forge
+```
+
+At the time of writing, the Python 3.14 migration is %x complete. This means that:
+
+- xx (x%) of packages that needed a rebuild were built for Python 3.14
+- xx (x%) have an open PR to get rebuilt for Python 3.14
+- xx (x%) of packages are still waiting for a dependency to be rebuilt.
+
+## What is already usable?
+
+While most binary packages needed to be rebuilt for Python 3.14,
+we did not need to rebuild packages that are `abi3`-compatible.
+Here, you can use the builds that were done with previous Python versions.
+Similarly, all pure Python packages (i.e. the ones we call `noarch: python` in the conda world) didn't need to be rebuilt and can be used directly.
+
+Still, as the statistics above show, not all packages are available for Python 3.14 yet.
+The most prominent here include `pytorch`, `numba`, and `cytoolz`.
+For the latter two, we need to wait for them to provide new releases that make them compatible with 3.14.
+In the case of PyTorch, we either need to wait for the new `2.10.0` release or may be able to backport changes to the `2.9.0` release.
+You can track the progress on the upstream side in [pytorch#156856](https://github.com/pytorch/pytorch/issues/156856) and on the conda-forge side in [pytorch-cpu-feedstock#420](https://github.com/conda-forge/pytorch-cpu-feedstock/pull/420).
+For `numba`, we expect the necessary changes to land in `0.63`.
+As a there is already [a beta](https://github.com/numba/numba/releases/tag/0.63.0b1) available, we will wait here for the final release and won't backport any changes.
+For other packages, similar approaches will be taken.
+You can track the progress for each of them if you click on [the respective link on the status page](https://conda-forge.org/status/migration/?name=python314).
+
+If you face any issue with the Python package itself, please report that on the [python-feedstock](https://github.com/conda-forge/python-feedstock/issues).
+For problems with individual packages, please do so on the respective `conda-forge/<package>-feedstock` repository.
+
+## How was conda-forge able to provide release day availability?
+
+Like we already did in the releases for [Python 3.13](https://conda-forge.org/blog/2024/09/26/python-313/) and [Python 3.12](https://conda-forge.org/news/2023/09/25/python-312-migration-and-python-311-by-default/), we have started building packages for Python 3.14 once the release candidate was available.
+As ABI-stability is guaranteed between release candidates, these packages are also usable with the final Python release.
```

Comment:
```suggestion
As ABI-stability is guaranteed between release candidates, these packages are also usable with the final Python release.
Unfortunately, as part of this release process, the [magic number stored in Python bytecode (`.pyc`) was bumped](https://discuss.python.org/t/python-3-14-0rc3-is-go/103815) in RC2 and RC3.
This means that the bytecode will be regenerated on the first run for packages built with a version earlier than RC3.
As the impact of this has been minor, we did not rebuild the respective packages and expect that through package updates, this will quickly become unnoticeable.
```

Reply:
Fine by me, though I would put it after the "Thus we were able"